src/app/app-routing.module.ts

```diff
@@ -1,8 +1,15 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { LoginComponent } from './login/login.component';
+import { RegisterComponent } from './register/register.component';
 
-
-const routes: Routes = [];
+// This routes the app from specific address bar inputs
+const routes: Routes = [
```

Comment:
This is awesome! Angular makes this so easy!

Reply:
They do once you know where to find it and how to use it... :)